src/components/browse/components/FacetList/RangeFacet.js

```diff
@@ -411,7 +411,7 @@ export class RangeFacet extends React.PureComponent {
             <div className={"facet range-facet" + (facetOpen ? ' open' : ' closed')} data-field={facet.field}>
                 <h5 className="facet-title" onClick={this.handleOpenToggleClick}>
                     <span className="expand-toggle col-auto px-0">
-                        <i className={"icon icon-fw icon-" + (isOpen ? "minus fas" : "plus fas")}/>
+                        <i className={"icon icon-fw icon-" + (facetOpen ? "minus fas" : "plus fas")}/>
```

Comment:
If had a value, but facet is closed, this icon wouldn't change (remain minus), so this addresses that. We could consider renaming variable 'isOpen' 🤔 lol.